### PR TITLE
updated a description for the docpot

### PR DESCRIPTION
**Pull request by Lucrezia-Cester**
This links to issue https://github.com/GSTT-CSC/hazen/issues/156. 

It attempts to give a description of each optional argument for the docpot

### CODE REVIEW

**Review by heyhaleema — APPROVED** [review]

File: hazenlib/__init__.py [hunk 1]
```diff
@@ -79,11 +79,14 @@
     hazen -h|--help
     hazen -v|--version
 Options:
+Report is an optional argument needed if you want to get a plot of your results.
+Calc_t1, calc_t, plate_number=<n>, show_template_fit, show_relax_fits, show_rois, verbose are optional argument for the function relaxometry.
```
Comment (heyhaleema) [inline hazenlib/__init__.py]:
```suggestion
'Calc_t1', 'calc_t', 'plate_number=<n>', 'show_template_fit', 'show_relax_fits', 'show_rois', 'verbose' are optional arguments for the relaxometry function.
```

File: hazenlib/__init__.py [hunk 2]
```diff
@@ -79,11 +79,14 @@
...
+Measured_slice_width is an optional argument for the function snr.
```
Comment (heyhaleema) [inline hazenlib/__init__.py]:
```suggestion
'Measured_slice_width' is an optional argument for the SNR function.
```

File: hazenlib/__init__.py [hunk 3]
```diff
@@ -79,11 +79,14 @@
...
+Log is an optional argument that allows to set the severity of the logs.
```
Comment (heyhaleema) [inline hazenlib/__init__.py]:
```suggestion
'Log' is an optional argument that allows users to set the severity of the logs.
```

**Review by heyhaleema — DISMISSED** [review]
@Lucrezia-Cester Looks good 👍 I've made some minor suggestions, but they're not hard requests. Equally happy for you to either commit or ignore these, so once the conversations are resolved, I'll merge.

**Comment by Lucrezia-Cester** [comment]
@heyhaleema, I have committed your changes

**Review by heyhaleema — APPROVED** [review]

**Pull request merged**